common/vospi.c

```diff
@@ -196,6 +196,11 @@ int vospi_deinit() {
     return omv_spi_deinit(&vospi.spi_bus);
 }
 
+int vospi_abort(void) {
+    vospi.flags &= ~VOSPI_FLAGS_CAPTURE;
```

Comment:
Shouldn't the spi transfer be aborted here?

Reply:
Yes, we could do that, but, then VOSPI has to reinit which can take a quite a while. Disabling the capture flag is enough to allow for frame buffer geometry changes and etc. Then vospi deinit() is still called on soft reboot. We do need to deinit though on CSI object destruction... whenever the framebuffer is deleted.

Reply:
Let me resend this along with other fixes.